MQuince/MQuince.Services.Implementation/DatePriorityStrategy.cs

```diff
@@ -0,0 +1,54 @@
+﻿using MQuince.Entities.Appointment;
+using MQuince.Entities.Users;
+using MQuince.Repository.Contracts;
+using MQuince.Services.Contracts.DTO.Appointment;
+using MQuince.Services.Contracts.DTO.Users;
+using MQuince.Services.Contracts.IdentifiableDTO;
+using MQuince.Services.Contracts.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MQuince.Services.Implementation
+{
+    public class DatePriorityStrategy : IStrategy
+    {
+        public RecommendAppointmentParameters _parameters;
+        public IAppointmentService _appointmentService;
+        public IDoctorService _doctorService;
+        public IDoctorRepository _doctorRepository;
+        public ISpecializationService specializationService;
+        private AppointmentService appointmentService;
+        private RecommendAppointmentParameters parameters;
```

Comment:
Private fields should start with _ followed by Lower Cammel Case notation
Consider making public fields private if you don't need them to be public.
Public fields should Upper Cammel Case notation